transport/x/redis/inbound.go

```diff
@@ -34,6 +34,10 @@ import (
 
 const transportName = "redis"
 
+const maxConnectRetries = 100
```

Comment:
You might consider making these parameters to the inbound, so the user can control them.